src/SAML2/XML/saml/AttributeStatement.php

```diff
@@ -85,6 +85,7 @@ public function decryptAttributes(XMLSecurityKey $key, array $blacklist = []): v
 
     /**
      * @return bool
+     * @return void
```

Comment:
Added this return type by accident?

Reply:
Must have been a rebase mistake.. Taking it out!